scripts/createlegends.js

```diff
@@ -273,7 +275,9 @@ let createLegendRamp = function(data, labels, lang, cb) {
   let styles = getStylesByResolution(data, styleType);
   let keys = Object.keys(styles);
   // availale res
-  let legendKey = keys.indexOf('res-max-50') !== -1 ? 'res-max-50' : keys[0];
+  //let legendKey = keys.indexOf('res-max-50') !== -1 ? 'res-max-50' : keys[0];
+  let legendKey = keys.indexOf('res-max-100') !== -1 ? 'res-max-100' : keys[0];
```

Comment:
We need to know why this `if ` exists otherwise we risk to break other styles. @loicgasser ?

Reply:
I verified all legends and it seems to work